File: ruby_small_problems/easy_6/2.rb
# takes an array of strings
# returns an array of the same string values,
# except with the vowels removed
#
# for each str in array,
# remove vowels (delete)
# put in new array
# return new array

VOWELS = 'aeiouAEIOU'

def remove_vowels(arr)
  arr.map{ |x| x.delete(VOWELS) }
end

p remove_vowels(%w(abcdefghijklmnopqrstuvwxyz))# == %w(bcdfghjklmnpqrstvwxyz)
p remove_vowels(%w(green YELLOW black white))# == %w(grn YLLW blck wht)
p remove_vowels(%w(ABC AEIOU XYZ))# == ['BC', '', 'XYZ']

File: 210/exercises/small_problems/string_and_text_processing/2.js
function removeVowels(strings) {
  return strings.map(word => word.replace(/[aeiou]/ig, ''));
}

console.log(removeVowels(['abcdefghijklmnopqrstuvwxyz']));
// ["bcdfghjklmnpqrstvwxyz"]
console.log(removeVowels(['green', 'YELLOW', 'black', 'white']));
// ["grn", "YLLW", "blck", "wht"]
console.log(removeVowels(['ABC', 'AEIOU', 'XYZ']));
// ["BC", "", "XYZ"]
